src/main/java/FastIO.java

```diff
@@ -0,0 +1,82 @@
+import java.io.*;
```

Comment:
Should list out all the import class instead of using wildcard.

Reply:
Thank you for your input! I will make the changes accordingly.